app/views/search/results.html.erb

```diff
@@ -10,46 +10,50 @@
   <%= render(partial: 'trigger_tacos') if tacos_enabled? %>
 
   <%= turbo_frame_tag "search-results", data: { turbo_action: "advance" } do %>
-    <div class="layout-3q1q layout-band">
-      <main id="results" class="col3q wrap-results">
-        <% if @show_primo_continuation %>
-          <%= render partial: 'search/primo_continuation' %>
-        <% elsif @results.present? && @errors.blank? %>
-          <h2 class="results-context"><%= results_summary(@pagination[:hits]) %></h2>
-          <p class="results-context-description">From all MIT Libraries sources</p>
-          <ol class="results-list" start="<%= @pagination[:start] %>">
-            <% case @active_tab %>
-            <% when 'primo' %>
-              <%= render(partial: 'search/result_primo', collection: @results, as: :result) %>
-            <% when 'timdex' %>
-              <%= render(partial: 'search/result', collection: @results, as: :result) %>
-            <% when 'all' %>
-              <% @results.each do |result| %>
-                <% if result[:source] == 'Primo' %>
-                  <%= render(partial: 'search/result_primo', locals: { result: result }) %>
-                <% else %>
-                  <%= render(partial: 'search/result', locals: { result: result }) %>
+    <div id="results-section">
+      <% if @show_primo_continuation %>
+        <%= render partial: 'search/primo_continuation' %>
+      <% elsif @results.present? && @errors.blank? %>
+      <h2 class="results-context"><%= results_summary(@pagination[:hits]) %></h2>
+      <p class="results-context-description">From all MIT Libraries sources</p>    
+      <div id="results-layout-wrapper">  
+        <main id="results">
+            <ol class="results-list" start="<%= @pagination[:start] %>">
+              <% case @active_tab %>
+              <% when 'primo' %>
+                <%= render(partial: 'search/result_primo', collection: @results, as: :result) %>
+              <% when 'timdex' %>
+                <%= render(partial: 'search/result', collection: @results, as: :result) %>
+              <% when 'all' %>
+                <% @results.each do |result| %>
+                  <% if result[:source] == 'Primo' %>
+                    <%= render(partial: 'search/result_primo', locals: { result: result }) %>
+                  <% else %>
+                    <%= render(partial: 'search/result', locals: { result: result }) %>
+                  <% end %>
                 <% end %>
               <% end %>
-            <% end %>
-          </ol>
-        <% elsif @errors.blank? %>
-          <div class="no-results">
-            <p class="hd-2">No results found for your search</p>
-          </div>
+            </ol>
+            <%= render partial: "results_callouts" %>
+          <% elsif @errors.blank? %>
+            <main id="results">
+              <div class="no-results">
+                <p class="hd-2">No results found for your search</p>
+              </div>
+            </main>
```

Comment:
This tag ends up closed twice in certain scenarios I believe.

I'd suggest realigning the if/elsif statements so it is easier to keep track of them. I suspect the line 45 closing `main` might need to move into a condition? Not sure though.